features/features.json

```
@@ -33,6 +33,13 @@
 		"urls": ["https://scratch.mit.edu/users/rgantzos/"],
 		"file": "advanced-search"
 	},
+	{
+		"title":"Searchable Dropdowns",
+		"description":"Easily search through options in the code editor. You can search for variables, costumes, and more!",
+	 	"credits":["Garbomuffin"],
```

if you're going to put my name here, at least format it correctly
```suggestion
	 	"credits":["GarboMuffin"],
```